package-lock.json

```diff
@@ -37337,6 +37274,17 @@
         "debug": "4.3.1",
         "openurl": "1.1.1",
         "yargs": "16.2.0"
+      },
+      "dependencies": {
+        "axios": {
```

Comment:
axios 0.21.4 is still included here

Reply:
i updated the override block to update the pointed dependencies